README.md

```diff
@@ -42,54 +8,26 @@ After you're finished please remove all the comments and instructions!
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
```

Comment:
- **[OPTIONAL]** I marked this requirement as optional but take in mind that the README file always must be professional. All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo for professional documentation. Please 🙏🏽 complete all the sections with the corresponding information about the current project.

| Expected | Your repo|
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/227753554-5e3c1749-8097-4315-b2dc-77ec12007c39.png) | ![image](https://user-images.githubusercontent.com/58642949/227753577-1e926da9-940e-4aa4-be52-36f6b2557203.png) |